src/amber/cli/templates/app/README.md.ecr

```diff
@@ -1,59 +1,34 @@
 # <%= @name %>
 
-<%= @name %> is a new project using [Amber Framework](https://amberframework.org/)
+This is a project written using [Amber Framework](https://amberframework.org/).
```

Comment:
How about "This project is based on Amber Framework" or "This project is powered by Amber Framework"?
This little tweak would avoid "This is a" at the beginning of the sentence, as well as avoid "written using", which could be understood as if Amber was used as a tool to "write" the project, which is not the case.